blog/2023-11-20-stately-studio-2-0/index.mdx

```diff
@@ -0,0 +1,146 @@
+
+[image]
+
+With Stately AI and Stately Sky together, you can go from an idea to a deployed workflow in minutes. The Stately editor helps you create much more flexible workflows than other workflow tools—you can use these workflows on the front end, back end, or both. Perfect for onboarding flows, collaborative apps, games, AI agents, and more. Stately Sky is also backed by Cloudflare Workers so that you can run your state machines in 200+ locations worldwide, with low latency and high availability. 
+
+You can [read more about Stately Sky in Gavin’s blog post](https://stately.ai/blog/2023-11-13-introducing-stately-sky).
+
+## Enhanced Stately editor
+
+Among all these huge new features, we have updated the Stately editor’s UI. We want [Stately.ai](http://stately.ai/) to feel as natural as a white-boarding or diagramming tool, while giving you the power to create robust state machines at any level of complexity.
+
+These updates significantly improve the editor’s performance, especially when working with large machines. We’ve also streamlined the editor with reorganized panels and a redesigned header and footer. The new design makes accessing the tools you need faster while also providing information about your machine, including its locked status, project visibility, and current version.
+
+### Targetless transitions
+
+As a part of our improved support for XState in the Stately editor, the editor now supports targetless transitions. Targetless transitions are useful when you want a transition on your root state but don’t want to keep resetting your machine to the initial state. They also open up the possibility for multiple target state support in the future, which is handy if you want to transition to simultaneous parallel states.
+
+### Zoom to selection
+
+You can now **zoom to selected** to re-center the canvas from the zoom menu or the right-click menu. This zoom is particularly helpful when working with large machines and demoing your machines to others, enabling you to jump to a different canvas area swiftly.
+
+### Generate summary
+
+As part of our Stately AI toolset for our Pro users, we now have an experimental **Get summary** option in the machine details panel. You can use this to create a summary of your state machine to make it even more understandable to non-technical stakeholders. We’ve also found it helpful in generating alternative text for images of our state machines. This feature could explain and summarize specific paths throughout the machine in the future, representing different use cases or edge cases.
+
+[image]
+
+## What’s coming next
+
+And soon, there will be even more! We have designed many updates to the Stately editor to support even more capabilities that are on the way. Including:
+
+- GitHub Sync
+- Inspection capabilities
+- System views and sequence diagrams for actors
+- Stately Sky analytics
+- XState version 5
+- Updated VSC Code extension
```

Comment:
Is it the right time to mention something to do with multi-player or at least multi-presence in the Editor?